app/views/pages/medicaid/amounts_self_employment.erb

```diff
@@ -0,0 +1,29 @@
+<div class="step-header">
+  <a href="/pages/medicaid/amounts_current_job" class="step-header__back-link">
+    <i class="button__icon icon-arrow_back"></i>
+  </a>
+  <h4 class="step-header__title">
+    Income & Expense Amounts
+  </h4>
+</div>
+
+<div class="form-card">
+  <header class="form-card__header">
+    <div class="form-card__title">Tell us more about the specifics of your self-employment income.</div>
+    <p class="text--help text--centered">
+      If you don't know the exact number, that's okay. These questions aren't required. However, your income determines the benefits you are eligible for.
```

Comment:
mind breaking lines at 80 chars? makes it easier to read text in GH!

Reply:
Yep!